Makefile

```diff
@@ -113,16 +112,33 @@ srpm: .srpm-done
 
 rpm: .rpm-done
 
-ubuntu-trusty:
-	cp packaging/ubuntu-trusty/ecs.conf ecs.conf
-	tar -czf ./amazon-ecs-init_${VERSION}.orig.tar.gz ecs-init ecs.conf scripts README.md
+ARCH:=$(shell uname -m)
+ifeq (${ARCH},x86_64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-v${VERSION}.tar
+else ifeq (${ARCH},aarch64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-arm64-v${VERSION}.tar
+endif
+
+/tmp/ecs-agent-version:
+	echo "${VERSION}" >/tmp/ecs-agent-version
+
+/tmp/ecs-agent.tar: /tmp/ecs-agent-version
+	curl -o /tmp/ecs-agent.tar ${AGENT_URL}
```

Comment:
Same here; don't use static filenames in /tmp

Reply:
done